install-k8s-perf.yml

```diff
@@ -1,3 +1,7 @@
+- hosts: all
```

Comment:
I don't we have group called all, need to add both master and worker

Reply:
Added a fix PR : 
https://github.com/ppc64le-cloud/k8s-ansible/pull/54